### PR TITLE
chore: lint for conventional commits

### LINKED ISSUE #2415: Use conventional commits
Use conventional commits to auto create changelog entries for each version.

Found a good documentation here: https://dev.to/xcanchal/monorepo-using-lerna-conventional-commits-and-github-packages-4m8m

### PR DESCRIPTION
**Pull request by thomas-krueger-axa**
Closes #2415

Fixes #

# Merge Checklist:
- [ ] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [ ] Tests are sufficient.
- [ ] Documentation is up to date.
- [ ] Changelog is up to date.
- [ ] Typescript typings for properties are up to date.
- [ ] Designers approved changes or no approval needed.
- [ ] Dependencies to other components still work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,11 @@ We are dedicated to building a welcoming, diverse, and safe community. We expect
 
 This repository is a monorepo managed by Lerna. This means that all components are centrally managed here, even though we publish them to NPM as separate packages.
 
+## Commits
+
+We are using [Conventional Commits](https://www.conventionalcommits.org/) to automatically versioning and creating changelog's.
```
Comment (markus-walther) [inline README.md]: ...to automatically **version and update changelogs.**

**Review by markus-walther — CHANGES_REQUESTED** [review]
Hi Thomas,

I could not see how this works in practice - can you include a commit message example and the changelog entry it automagically creates in the README?

Other than that, just a wording fix is needed.

Cheers, Markus

**Review by amir-ag — DISMISSED** [review]

**Review by marcwelz — DISMISSED** [review]
Changes requested from @markus-walther needs to be adressed, otherwise LGTM

**Review by marcwelz — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -74,6 +74,20 @@ We are dedicated to building a welcoming, diverse, and safe community. We expect
 
 This repository is a monorepo managed by Lerna. This means that all components are centrally managed here, even though we publish them to NPM as separate packages.
 
+## Commits
+
+We are using [Conventional Commits](https://www.conventionalcommits.org/) to automatically version the components and update their changelog's.
```
Comment (markus-walther) [inline README.md]: grammar: changelogs ('s is genitive singular, but you want plural).

File: README.md [hunk 3]
```diff
@@ -74,6 +74,20 @@ We are dedicated to building a welcoming, diverse, and safe community. We expect
...
+Feel free to use a [tool of your choice](https://www.conventionalcommits.org/en/about/#tooling-for-conventional-commits) to generate these commits.
+
+Example for a commit message
```
Comment (markus-walther) [inline README.md:82]: Good!

File: README.md [hunk 4]
```diff
@@ -74,6 +74,20 @@ We are dedicated to building a welcoming, diverse, and safe community. We expect
...
+```
+feat(button): add new color blue
+
+Closes #ticket-1234
```
Comment (markus-walther) [inline README.md:86]: Hmmm.. does this force multi-line commit messages??? I use git CLI in a shell and never do multi-line....

File: README.md [hunk 5]
```diff
@@ -74,6 +74,20 @@ We are dedicated to building a welcoming, diverse, and safe community. We expect
...
+```
+
+For more examples, especially with BREAKING CHANGES, have a look at the [Conventional Commits Examples](https://www.conventionalcommits.org/en/v1.0.0/#examples).
+
```
Comment (markus-walther) [inline README.md:90]: I asked for a second thing: show the automatic Changelog entry resulting from a conventional commit. Seems you forgot this part, did you?

Comment (markus-walther) [reply README.md:90]: I thought one of the main motivations for conventional commits in our case here is to automate changelog entries... so an example would be highly relevant.

**Review by markus-walther — CHANGES_REQUESTED** [review]
Hi Thomas,

good progress. Some findings remain, sorry ;-)

Cheers, Markus

File: README.md (hunk 5 above)
Comment (thomas-krueger-axa) [reply README.md:90]: It depends on the library which we wanna use to create Changelogs. So for now, we only want to force the Commits, later the Changelogs. Means I cannot give you an Example of an Changelog entry.

File: README.md (hunk 4 above)
Comment (thomas-krueger-axa) [reply README.md:86]: Yes it does. But should not be a problem. This is what I found on Google.

>Another method of adding a multi-line Git commit message is using quotes with your message, though it depends on your shell's capacity. To do this, add single or double quotes before typing the message, keep pressing enter and writing the next line, and finally close the quote at end of the message.

Comment (markus-walther) [reply README.md:86]: OK, understood

File: README.md (hunk 5 above)
Comment (markus-walther) [reply README.md:90]: I see.

**Review by markus-walther — APPROVED** [review]
Thomas,

you answered everything well, so LGTM.

Cheers, Markus

**Pull request merged**